### PR TITLE
Pause all on init

### PR DESCRIPTION
**Pull request by olga24912**

### CODE REVIEW

File: eth-connector-tests/src/connector.rs [hunk 1]
```diff
@@ -56,6 +56,7 @@ async fn test_ft_transfer() {
 #[tokio::test]
 async fn test_ft_transfer_user() {
     let contract = TestContract::new().await.unwrap();
+    contract.unpause_all().await;
```
Comment (karim-en) [inline eth-connector-tests/src/connector.rs:59]: Let's combine the new and unpause in one helper function instead of duplicate this action

Comment (aleksuss) [inline eth-connector-tests/src/connector.rs:59]: Does moving this call to the `TestContract::new` method make sense?

File: eth-connector/src/lib.rs [hunk 2]
```diff
@@ -253,6 +253,9 @@ impl EthConnectorContract {
 
         this.acl_init_super_admin(env::predecessor_account_id());
         this.acl_grant_role("DAO".to_string(), owner_id.clone());
+        this.acl_grant_role("PauseManager".to_string(), env::predecessor_account_id());
```
Comment (karim-en) [inline eth-connector/src/lib.rs:256]: The DAO role is enough to do pause and unpause

Comment (karim-en) [reply eth-connector/src/lib.rs:256]: Btw, what is the owner_id and why we need to pass it on init?

Comment (olga24912) [reply eth-connector/src/lib.rs:256]: But `env::predecessor_account_id()` doesn't have DAO role

File: eth-connector-tests/src/connector.rs (hunk 1 above)
Comment (olga24912) [reply eth-connector-tests/src/connector.rs]: fixed: https://github.com/Near-One/aurora-eth-connector/pull/88/commits/9c3373aeeb4b3cbe5a1d2da4e4da254482105e49

Comment (olga24912) [reply eth-connector-tests/src/connector.rs]: fixed: https://github.com/Near-One/aurora-eth-connector/pull/88/commits/9c3373aeeb4b3cbe5a1d2da4e4da254482105e49

File: eth-connector/src/lib.rs (hunk 2 above)
Comment (olga24912) [reply eth-connector/src/lib.rs:256]: If we can init the EthConnector by using DAO we can remove owner_id. But I am not sure, that we can: metadata can be too big to be signed by Ledger.

Also I am not sure, that it is good idea to give the predecessor_account_id the super admin role. 

I need to give to predecessor_account_id DAO or PauseManager role, but I can remove it after pausing....

Comment (mrLSD) [reply eth-connector/src/lib.rs:256]: @olga24912 could you a bit clarify why we don't need `owner_id` for DAO?

**Review by karim-en — APPROVED** [review]

**Review by mrLSD — APPROVED** [review]

**Pull request merged**